concent_api/api_testing_common.py

```diff
@@ -375,16 +368,24 @@ def create_signed_task_to_compute(
     deadline: int,
     timestamp: Optional[Union[datetime.datetime, str]]=None,
     provider_public_key: Optional[bytes]=None,
+    provider_private_key: Optional[bytes]=None,
     requestor_public_key: Optional[bytes]=None,
-    requestor_ethereum_public_key: Optional[bytes]=None,
-    requestor_ethereum_private_key: Optional[bytes]=None,
-    provider_ethereum_public_key: Optional[bytes]=None,
+    requestor_private_key: Optional[bytes]=None,
     want_to_compute_task: Optional[WantToComputeTask] = None,
     price: int=1,
     size: int=1,
     package_hash: str='sha1:57786d92d1a6f7eaaba1c984db5e108c68b03f0d',
     script_src: Optional[str]=None,
 ) -> TaskToCompute:
+    # Temporary workaround for requestor's and provider's keys until all Concent use cases will have payments
+    # When them will have payments then all keys will be taken from SCIBaseTest class
+    if provider_public_key is None and provider_private_key is None:
```

Comment:
@pawelkisielewicz It has to be done this way, because we have to be sure that pairs of keys (private and public) must be correct. If we change it and do it your way someone could call this function for example like this:
```python
create_signed_task_to_compute(
...
provider_public_key=new_key,
...
```
and it will create message with new public_key which won't be related to provider's private_key which is stored as `PROVIDER_PRIVATE_KEY` and it leads us to bug :)